### PR TITLE
optimization-for-dashboard-license

### PR DESCRIPTION
**Pull request by abby-cyber**

### CODE REVIEW

**Review by whitewum — APPROVED** [review]

File: docs-2.0/nebula-dashboard-ent/11.dashboard-ent-license.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+A license is a software authorization certificate used to authorize the use of a software product. When deploying Nebula Dashboard Enterprise Edition, you need to add a license to start it. This document describes the license information on Nebula Dashboard Enterprise Edition.
+
+## Precautions
+
+- If the license file is not deployed, Nebula Dashboard Enterprise Edition cannot be started.
+
+- Do not modify the license file, otherwise the license will become invalid.
+
+- If the license is about to expire, send email to `inquiry@vesoft.com` to apply for renewal.
```
[the remark this answers is not in the document]
Comment (abby-cyber) [reply docs-2.0/nebula-dashboard-ent/11.dashboard-ent-license.md:11]: > 

hmmm email is uncountable, will keep it.

File: docs-2.0/nebula-dashboard-ent/11.dashboard-ent-license.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  - If you start the Enterprise Edition within 30 days before the license expires or on the day the license expires, a log will be printed as a reminder.
+
+  - The license can still be used for 14 days after it expires.
+
+  - If the license has expired for 14 days, you will not be able to start the Enterprise Edition, and a log will be printed as a reminder.
+
+
+## Obtain a Nebula Dashboard Enterprise Edition license
+
+Send email to `inquiry@vesoft.com` to apply for a Nebula Dashboard Enterprise Edition license.
```
[the remark this answers is not in the document]
Comment (abby-cyber) [reply docs-2.0/nebula-dashboard-ent/11.dashboard-ent-license.md:24]: > 

ditto

[the remark this answers is not in the document]
Comment (abby-cyber) [reply docs-2.0/nebula-dashboard-ent/11.dashboard-ent-license.md:24]: > 

ditto

[the remark this answers is not in the document]
Comment (abby-cyber) [reply docs-2.0/nebula-dashboard-ent/11.dashboard-ent-license.md:24]: > 

ditto

**Review by cooper-lzy — APPROVED** [review]

**Pull request merged**